Apps/Playground/Win32/App.cpp

```diff
@@ -80,7 +80,7 @@ namespace
         }
         auto width = rect.right - rect.left;
         auto height = rect.bottom - rect.top;
-        runtime = std::make_unique<Babylon::RuntimeWin32>(hWnd, rootUrl, width, height);
+        runtime = std::make_unique<Babylon::RuntimeWin32>(hWnd, rootUrl, static_cast<float>(width), static_cast<float>(height));
```

Comment:
Might wanna merge from master which looks like this: https://github.com/BabylonJS/BabylonNative/blob/master/TestApp/Source/Win32/App.cpp#L84-L86